packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -226,6 +227,8 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
             versions: protocolVersions,
             nonce: uuid(),
             epoch: epochTracker.fluidEpoch,
+            relayUserAgent: client.details.environment !== undefined ?
+                `${client.details.environment};driverVersion:${pkgVersion}` : `driverVersion:${pkgVersion}`,
```

Comment:
space after ; would be nice - a bit more readable (by humans, though requires trimming by bots :))